### DISCUSSION TITLE
Creation category Item

### DISCUSSION THREAD

**Discussion by Jackman97** [Mezryss/FVTT-Genesys#176] [Q&A] [answered]
Sorry if this is a really stupid question.
I've been playing around with item creation and I'd like to create item categories, actions that would be like rolls with skills and difficulties set for either spells or skills, but I can't find the file responsible for creating that item category in a working foundry.
What files are in charge of item categories?
I've been looking in several places but I haven't found it.

**Comment by Assembling-Kings** [comment, marked as answer]
Hmmm, I quite don't understand what you're asking so I'll post a few things in hope one of these is related to what you asked.
- If you're referring to standard system use, then to create an item click on the Create Item button of the Items tab. It should bring up a popup with the list of possible items you can create.
- If you are asking about adding new item types then here's an example of the places that you need to touch to create one from scratch [#b2d41e8](https://github.com/Mezryss/FVTT-Genesys/commit/b2d41e8c4bfbaa935c0c96b4cc10c89efbe7fd1f)
- If you are talking about active effects, then you can look at this PR instead #152
